packages/marqus-desktop/src/renderer/App.tsx

```diff
@@ -186,7 +186,7 @@ export async function loadInitialState(
       },
       editor: {
         ...ui.editor,
-        tabs: tabs.map(t => ({ ...t, fromPreviousSession: true })),
```

Comment:
`fromPreviousSession` was an old flag that we used to use when determining what tabs to open on the first render. It's not used anywhere anymore so it's been removed.